.papr.yml

```diff
@@ -19,9 +30,10 @@ packages:
   - openssl-devel
   - redhat-rpm-config
 
-context: 'fedora/25/atomic | origin/v3.6.0-alpha.1'
+context: 'fedora/25/atomic | origin/v3.6.0-alpha.2'
```

Comment:
Is it necessary to have the version in the checkmark text? How about `context: fedora/25/atomic`? If one wants to know the version, it will be in the config and should be in the job logs perhaps, WDYT?